src/deadline/job_attachments/_glob.py

```diff
@@ -0,0 +1,40 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+
+import json
+from pathlib import Path
+from deadline.client.exceptions import NonValidInputError
+from deadline.job_attachments.models import GlobConfig
+
+
+def _process_glob_inputs(glob_arg_input: str) -> GlobConfig:
```

Comment:
Since we have GlobConfig class, can this be a function there?

Reply:
No, because GlobConfig is a `@dataclass`. I don't mind moving the dataclass to this file though.

But then model classes are all in models. Both works, preferences?

Reply:
What about GlobConfig dataclass in models, and have a normal class with GlobConfig as class variable for the functions?